.github/codecov.yml

```diff
@@ -16,7 +16,7 @@ coverage:
     project:
       default:
         target: 50%
-        threshold: 0.05 # allow this much decrease on project
+        threshold: 5% # allow this much decrease on project
```

Comment:
question: why?

Reply:
I noticed that I was getting a lint warning, and that the code cov documentation used percentages instead of decimals as well.